Library/Homebrew/cmd/cat.rb

```diff
@@ -25,6 +26,17 @@ def cat
     raise "`brew cat` doesn't support multiple arguments" if args.remaining.size > 1
 
     cd HOMEBREW_REPOSITORY
-    safe_system "cat", formulae.first.path, *Homebrew.args.passthrough
+    pager = "cat"
+    unless ENV["HOMEBREW_BAT"].nil?
+      name = "bat"
+      unless Formula[name].installed?
+        fi = FormulaInstaller.new(Formula[name])
+        fi.prelude
+        fi.install
+        fi.finish
+      end
+      pager = "#{Homebrew::DEFAULT_PREFIX}/bin/#{name}"
```

Comment:
`pager = bat` seems sufficient?

Reply:
Not on linux at least. It can't find `bat` without full path.